obsidian/_pages/about.md

```diff
@@ -27,5 +27,5 @@ I spoke at RubyConfs, Euruko, universities, and more. I believe [good writing is
 
 Some personal texts:
 
-- [Check out my work setup]({% link obsidian/_pages/my-digital-workbench.md %})
-- [How to use Jekyll with Obsidian.md]({% link obsidian/guide/_posts/2022-10-15-jekyll-with-obsidian.md %})
\ No newline at end of file
+- [Check out my work setup](./my-digital-workbench.md)
```

Comment:
I removed any regex I have locally to avoid transformations, and note how `_pages` is removed in that case as well.

<img width="476" alt="Screenshot 2025-01-19 at 19 49 31" src="https://github.com/user-attachments/assets/79c23223-6368-43fe-9b6b-404f44df188d" />